recipes/onnxruntime/all/conanfile.py

```diff
@@ -150,7 +153,7 @@ def generate(self):
         tc.variables["onnxruntime_TVM_CUDA_RUNTIME"] = False
         tc.variables["onnxruntime_TVM_USE_HASH"] = False
         tc.variables["onnxruntime_CROSS_COMPILING"] = False
-        tc.variables["onnxruntime_DISABLE_CONTRIB_OPS"] = False
+        tc.variables["onnxruntime_DISABLE_CONTRIB_OPS"] = self.options.with_cuda_provider
```

Comment:
Does this not work when CUDA is enabled?

Reply:
See original post: I have build issues when CUDA is enabled (the package works fine without it being enabled).
It can probably be fixed, but it is out of scope for me currently and I figured - since it is perfectly backwards compatible - this is 'good enough' as an improvement already.

I'll see whether I find the time later on to fix it, but even if I do not, this will help others get forward, even if they do need contrib.

Reply:
Thanks for the extra insight @bverhagen :)